lib/stats.js

```diff
@@ -179,7 +176,7 @@ Stats.prototype.wire = function wire(server) {
 
     doPublish("version", version);
     doPublish("started_at", server.stats.started.toISOString());
-    doPublish("uptime", mom.from(Date.now(), true));
+    doPublish("uptime", Math.ceil((Date.now() - this.started) / 1000) + ' seconds');
```

Comment:
@mcollina I think we just need to replace `this` with `server.stats` to pass the failing test.